app/views/users/_user_table.html.erb

```diff
@@ -3,12 +3,15 @@
     <thead>
       <tr>
         <th>Name</th>
+        <% if params[:translator] %>
+        <th> Known Languages </th>
+        <% end %>
         <th>Duties</th>
         <th>Phone</th>
         <th>Location</th>
         <% if params[:foster_mentor] == 'true' %>
           <th>Mentees</th>
-        <% else %>
+        <% elsif !params[:translator] %>
```

Comment:
```_user.html.erb``` needs a complimentary if statement as the current_foster_dogs and coordinated_dogs values still show up.  

https://cl.ly/0H21473G2d0j